addons/spectate/functions/fnc_updateInfo.sqf

```diff
@@ -81,8 +81,20 @@ if (GVAR(uiVisible) && GVAR(showInfo)) then {
             };
             nil
         } count _bodyInfo; // count used here for speed, ensure nil above this line
+
+        private _bloodVol = _unit getVariable [QACEGVAR(medical,bloodVolume), 6];
+        private _pain = _unit getVariable [QACEGVAR(medical,pain), 0];
+        private _painSuppress = _unit getVariable [QACEGVAR(medical,painSuppress), 0];
+        private _heartRate = _unit getVariable [QACEGVAR(medical,heartRate), -1];
```

Comment:
May look funky in edge cases with default of -1

Reply:
I'm thinking if one were to click a spectator unit from the left bar and view its stats. Not blocking, just odd